web-frontend/src/pages/Assets/Assets.tsx

```diff
@@ -1,15 +1,108 @@
-import Placeholder from '@/layouts/Placeholder/Placeholder';
 import { WalletLayout, MenuItem } from '@/layouts/WalletLayout/WalletLayout';
 import Intl from '@/i18n/i18n';
-import { FiDisc } from 'react-icons/fi';
+import { FiPlus } from 'react-icons/fi';
+import {
+  Button,
+  Input,
+  MassaLogo,
+  Mrc20,
+  Mrc721,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  Token,
+} from '@massalabs/react-ui-kit';
+import { useState } from 'react';
+
+// TODO : Connect to /api/assets
 
 function Assets() {
+  const [modal, setModal] = useState(false);
+
+  // mock token list data
+  const massaToken = {
+    logo: <MassaLogo size={40} />,
+    formattedBalance: '000,000,000.00',
+    name: 'Massa',
+    symbol: 'SYMBL',
+    rawBalance: '000,000,000.000000000',
+  };
+
+  const mrc20Token = {
+    logo: <Mrc20 size={40} />,
+    formattedBalance: '000,000,000.00',
+    name: 'Mrc20',
+    symbol: 'SYMBL',
+    rawBalance: '000,000,000.000000000',
+  };
+
+  const mrc721Token = {
+    logo: <Mrc721 size={40} />,
+    formattedBalance: '000,000,000.00',
+    name: 'Mrc721',
+    symbol: 'SYMBL',
+    rawBalance: '000,000,000.000000000',
+  };
+
+  const tokenList = [massaToken, mrc20Token, mrc721Token];
+
   return (
     <WalletLayout menuItem={MenuItem.Assets}>
-      <Placeholder
-        message={Intl.t('placeholder.teaser-assets-page')}
-        icon={<FiDisc size={114} />}
-      />
+      <div className="flex flex-col bg-secondary p-10 h-fit w-2/3 rounded-lg">
```

Comment:
```suggestion
      <div className="bg-secondary p-10 h-fit w-2/3 rounded-lg">
```

I think you don't need to put flex here. Can you test it for me ? 🤔